Software/GuitarPedal/Effect-Modules/reverb_delay_module.cpp

```diff
@@ -539,18 +539,20 @@ void ReverbDelayModule::SetTempo(uint32_t bpm) {
     UpdateLEDRate();
 }
 
-float ReverbDelayModule::GetBrightnessForLED(int led_id) {
+float ReverbDelayModule::GetBrightnessForLED(int led_id) const {
     float value = BaseEffectModule::GetBrightnessForLED(led_id);
 
     float osc_val = led_osc.Process();
```

Comment:
I didn't really know how to refactor this/want to test it so I just made led_osc mutable since it seems to only be used for the LED